ChangeLog.md

```diff
@@ -1,6 +1,7 @@
 2017.09 - version 0.13.0-preview
 
 ALL
+* Removed Nokogiri from Gemfile because it causes bundler fail to install azure-storage. Added it back to runtime dependency and explicitly require user to install the correct version of Nokogiri in README.md.
 * Service version is upgrade to 2016-05-31.
```

Comment:
... upgraded ...

Reply:
....Sure, don't know why this is missed...